pytorch_toolkit/image_classification/model_templates/custom-classification/README.md

```diff
@@ -1,15 +1,80 @@
 # Image classification
 
-Models that are able to classify images.
+Performance results pivot table:
 
-| Model Name | Complexity (GFLOPs) | Size (Mp) | Top-1 on ImageNet (%) | Links |
-| ---        | ---                 | ---       | ---                   | ---                   |
-| MobilenetV2 1.0 | 0.66           | 3.50      | 73.0                  | [model template](./mobilenet_v2_w1/template.yaml)
+| Model Name | Complexity (GFLOPs) | Size (Mp) | AVG mAP (%) | AVG Top-1 (%) | AVG Top-5 (%) | Links |
+| ---        | ---        | ---        | ---        | ---        | ---        | ---        |
+| EfficientNet b0 | 0.76 | 4.14 | 92.267 | 90.156 | 98.283 | [model template](./efficientnet_b0/template.yaml) |
```

Comment:
links to snapshots are also useful so user can easily see that pretrained model is available (see https://github.com/openvinotoolkit/training_extensions/blob/develop/pytorch_toolkit/object_detection/model_templates/face-detection/readme.md)

Reply:
added links to pretrained weights